src/content/app/genome-browser/components/browser-location-indicator/BrowserLocationIndicator.tsx

```diff
@@ -51,7 +52,11 @@ export const BrowserLocationIndicator = (props: Props) => {
     <div className={className}>
       <div className={styles.chrLabel}>Chromosome</div>
       <div className={styles.chrLocationView} {...onClickProps}>
-        <div className={styles.chrCode}>{chrCode}</div>
+        {chrCode === 'Chromosome' ? (
```

Comment:
You need to check the `isCircular` field from the karyotype response to check whether the chromosome should be circular. `chrCode` here is just the chromosome name, which should be used to check the response from the karyotype endpoint:

![image](https://user-images.githubusercontent.com/6834224/150967928-ec8a3597-de44-478c-b206-bb445df13cfc.png)

Reply:
Aah gotcha. Will update it soon